src/components/middle/immedia/Immedia.tsx

```diff
@@ -294,7 +294,7 @@ const Immedia: FC<OwnProps & StateProps> = ({ chatId, currentUser }) => {
               canvas.width,
               canvas.height,
             );
-            const image = canvas.toDataURL('image/jpeg');
+            const image = canvas.toDataURL();
```

Comment:
Why revert the jpeg change?

Reply:
Same behaviour as the webogram implementation, using jpeg can reduce even further the size but it's not necessary.